FirebaseAuthUI/FUIAuth.m

```diff
@@ -31,6 +31,18 @@
 #import "FUIPasswordSignInViewController_Internal.h"
 #import "FUIPasswordVerificationViewController.h"
 
+/** @typedef EmailHintSignInCallback
+    @brief The type of block invoked when an emailHint sign-in event completes.
+
+    @param authResult Optionally; Result of sign-in request containing both the user and
+       the additional user info associated with the user.
+    @param error Optionally; the error which occurred - or nil if the request was successful.
+    @param credential Optionally; The credential used to sign-in.
+ */
+typedef void (^EmailHintSignInCallback)(FIRAuthDataResult *_Nullable authResult,
```

Comment:
Should this be `FUIEmailHintSignInCallback`?

Reply:
I didn't think we need the prefix here since it's an internal callback.